site/src/components/ActiveUserChart/ActiveUserChart.tsx

```diff
@@ -120,11 +161,13 @@ type ActiveUsersTitleProps = {
 export const ActiveUsersTitle: FC<ActiveUsersTitleProps> = ({ interval }) => {
 	return (
 		<div css={{ display: "flex", alignItems: "center", gap: 8 }}>
-			{interval === "day" ? "Daily" : "Weekly"} Active Users
+			{interval === "day" ? "Daily" : "Weekly"} User Activity
 			<HelpTooltip>
 				<HelpTooltipTrigger size="small" />
 				<HelpTooltipContent>
-					<HelpTooltipTitle>How do we calculate active users?</HelpTooltipTitle>
+					<HelpTooltipTitle>
+						How do we calculate user activity?
+					</HelpTooltipTitle>
 					<HelpTooltipText>
 						When a connection is initiated to a user&apos;s workspace they are
```

Comment:
Since we’re updating the data on this chart, should we also update the content of this tooltip?